### DISCUSSION TITLE
Change the name in Command Palette to ionide or fsharp (better results for searching)

### DISCUSSION THREAD

**Discussion by sharno** [ionide/ionide-vscode-fsharp#1634] [Ideas]
VSCode mostly shows `F` results when searching for `F#` in the command palette. It also feels a lot easier to write `fsharp` or `ionide` to reach quickly for a command from the extension.

The photo shows that only the `F` is being highlighted in the higher results. Also showing that there are many results before the actual extension commands.
![image](https://user-images.githubusercontent.com/601606/134602899-2d3c96ac-39db-4d67-bc94-b2ca9df75b17.png)

[a comment by an automated account is not shown]

**Comment by baronfel** [comment]
That's not a bad idea. it wouldn't be hard either, just changing a bunch of the `contributes.commands` entries.

**Comment by Krzysztof-Cieslak** [comment]
I'm not really sure if I am comfortable with this change - it's a breaking change for the users, it's a longer thing to type, and it's not as obvious as F#. I think a better solution here would be complaining about this issue to the VSCode team. :-)

**Comment by sharno** [comment]
- I think actually that this is an additive change. It doesn't really break any commands that you write because of the fuzzy search. 
- Second is that because VSCode doesn't really consider the `#` sign in the search, so typing `f: run` is equal to typing `f#: run` as you see in the photo it's matching everything including the `#` sign. 
- Lastly, the `#` sign is really not natural to type while developing in F# and it needs a key combination, so I it doesn't feel really natural while exploring/using the extension to use a key combination while typing in the search bar. 

Did I manage to convince you? 😄 
(I think even if we file an issue with VSCode itself I'd still prefer to change this in Ionide)

![image](https://user-images.githubusercontent.com/601606/134689678-72edbfa9-aeff-42b2-a4bd-c50a6bb927d8.png)


![image](https://user-images.githubusercontent.com/601606/134689599-2edca47e-ec2c-4ce5-ad04-d8d202e4c623.png)

**Comment by HerrNiklasRaab** [comment]
I agree with @sharno. I am in favor of the name "Fsharp".

**Comment by Krzysztof-Cieslak** [comment]
Converting this to the discussion rather than issue.

**Comment by Krzysztof-Cieslak** [comment]
This has been fixed on the VSCode side, and the command palette behaves as expected.
<img width="536" alt="image" src="https://user-images.githubusercontent.com/5427083/180610162-3a022863-6188-40d3-8921-a9845fab1ff2.png">
